src/engine/topology.ts

```diff
@@ -357,7 +357,7 @@ export abstract class Layer extends serialization.Serializable {
   protected _trainableWeights: LayerVariable[];
   private _nonTrainableWeights: LayerVariable[];
   private _losses: RegularizerFn[];
-  private _updates: TensorInterface[];
+  private _updates: Tensor[];
```

Comment:
I think this is still correct, but it is more restrictive than what we had before.